app/models/like.rb

```diff
@@ -0,0 +1,9 @@
+class Like < ApplicationRecord
+  belongs_to :user, class_name: 'User', foreign_key: 'author_id'
+  belongs_to :post, class_name: 'Post', foreign_key: 'post_id'
+
+  def self.update_post_likes_count(post)
```

Comment:
### Optional
- How about using the `after_save` rails helper method to make the model take the action of updating the likes counter each time the model is being saved to the database!